config/thoth.yaml

```diff
@@ -421,13 +421,6 @@ repositories:
           labels: [bot]
       - name: info
 
-  - slug: thoth-station/graph-sync-operator
```

Comment:
Nice catch!

The repo was renamed [1] . Could you change it to:

```yaml
  - slug: thoth-station/graph-sync-scheduler
```
[1] https://github.com/thoth-station/graph-sync-scheduler

Reply:
https://github.com/thoth-station/kebechet/blob/a485cadadd4e91ce669c1dd8faf4575f786806a1/config/thoth.yaml#L201
It already exists I guess?

Reply:
:+1: